d2.config.js

```diff
@@ -11,7 +11,9 @@ const config = {
     pwa: {
         enabled: true,
         caching: {
-            patternsToOmitFromAppShell: [/.*/],
+            patternsToOmitFromAppShell: [
+                /^(?!.*me\/authorization)(?!.*me\?fields).*$/,
```

Comment:
I needed to add a couple of exceptions to have all types of maps work:
`/^(?!.*me\/authorization)(?!.*me\?fields)(?!.*systemSettings)(?!.*analytics).*$/`
I am not too sure about `analytics` but without it I had only the first map of the "Cases Malaria" loading.

Reply:
I think the RE for `analytics` is a bit too generic here, it would match all requests on our test instances because they contain `analytics` in the path).
If we need to add it we should perhaps use this instead:
```suggestion
                /^(?!.*me\/authorization)(?!.*me\?fields)(?!.*systemSettings)(?!.*analytics\?).*$/,
```